.github/workflows/dep_build.yml

```diff
@@ -17,7 +17,9 @@ jobs:
       matrix:
         java_version: ['8', '17', '21']
         # Versions need to align with ones in 'main.yml' workflow
-        kotlin_version: ['1.7.22', '1.8.10', '1.9.23', '2.0.0-Beta4']
+        # kotlin-reflect 1.8.2x has a bug and some tests fail, so we are downgrading to 1.8.10.
+        # https://youtrack.jetbrains.com/issue/KT-65156
+        kotlin_version: ['1.8.10', '1.9.23', '2.0.0-Beta4']
```

Comment:
Presumably 2.0.0-Beta4 could become `2.0.0` (or `2.0.0-Final` or whatever it is?)

Reply:
It has been corrected by another PR.
https://github.com/FasterXML/jackson-module-kotlin/tree/2.18/.github/workflows